tests/data/RTL_kerning_v3.glyphs

```diff
@@ -0,0 +1,241 @@
+{
+.appVersion = "3108";
+.formatVersion = 3;
+DisplayStrings = (
+"אב",
+"/hah-ar.init ر/hah-ar.init.swsh"
+);
+axes = (
+{
+name = Weight;
+tag = wght;
+}
+);
+date = "2022-02-09 20:07:11 +0000";
+familyName = "RTL kerning";
+fontMaster = (
+{
+axesValues = (
+100
+);
+id = m01;
+metricValues = (
+{
+pos = 800;
+},
+{
+pos = 700;
+},
+{
+pos = 500;
+},
+{
+},
+{
+pos = -200;
+},
+{
+}
+);
+name = Regular;
+}
+);
+glyphs = (
+{
+glyphname = A;
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 65;
+},
+{
+glyphname = B;
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 66;
+},
+{
+glyphname = C;
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 67;
+},
+{
+glyphname = "hah-ar.init";
+kernRight = "hah-ar.init";
+lastChange = "2022-02-21 17:28:46 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+},
+{
+glyphname = "hah-ar.init.swsh";
+kernRight = "hah-ar.init.swsh";
+lastChange = "2022-02-21 17:28:56 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+},
+{
+glyphname = "reh-ar";
+kernLeft = "reh-ar";
+lastChange = "2022-02-21 17:28:42 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 1585;
+},
+{
+glyphname = "alef-hb";
+kernLeft = leftAlef;
+lastChange = "2022-02-21 14:15:31 +0000";
+layers = (
+{
+layerId = m01;
+width = 700;
+}
+);
+metricLeft = "=50";
+metricRight = "=50";
+unicode = 1488;
+},
+{
+glyphname = "bet-hb";
+kernRight = rightBet;
+lastChange = "2022-02-21 11:18:27 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 1489;
+},
+{
+glyphname = "he-hb";
+lastChange = "2022-02-17 09:36:21 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 1492;
+},
+{
+glyphname = "one-ar";
+kernLeft = "left-one-ar";
+lastChange = "2022-02-21 14:41:31 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 1633;
+},
+{
+glyphname = "one-ar.wide";
+kernLeft = "left-one-ar";
+lastChange = "2022-02-21 14:56:37 +0000";
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+},
+{
+glyphname = one;
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 49;
+},
+{
+glyphname = space;
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 32;
+}
+);
+instances = (
+{
+axesValues = (
+100
+);
+instanceInterpolations = {
+m01 = 1;
+};
+name = Regular;
+}
+);
+kerningRTL = {
+m01 = {
+"@MMK_R_leftAlef" = {
```

Comment:
is it right ("MMK_R"), is it left ("leftAlef"), first, second?! My mind just exploded 🤯

Reply:
perhaps I get it now, please confirm. In the Glyphs3's `kerningRTL` dictionary, if the _first_ key is a group it's prefixed `@MMK_R_`, whereas the second key in the (nested) dict, is prefixed `@MMK_L_` if a group. Hence, in RTL kerning, the (logical) first glyph is the one placed on the right-hand side, the second glyph is the one on the left-hand side. The opposite is true for LTR kerning, where first is left and second is right. All good so far.

Now, the meaning of "left" and "right" in the `leftKerningGroup` and `rightKerningGroup` glyph properties is _not_ "the group this glyph belong when it occurs in the left|right position", but rather the _side_ or _direction_ towards which this glyph is kerned against: so, for RTL kerning (like your `leftAlef` above), "left" means the group pertains to the left (in visual sense) side of the alef glyph, when this is placed in the first (logical) position next to second glyph, whose right (visual) side is the one that participate in the kerning...

Reply:
There are several stages where the kerning data is accessed. For me the more important part is during editing the font data in the UI, not the generating of the feature code. 
I think the confusion comes form the fact that the classes are defined on a single glyph. The classes (left/right/top/bottom) collects all glyphs that are similar on that side. The kerning is defined on pairs, so here the left/right mean the glyph on that side of the pair. The right side of the left glyph is kerned to the left side of the right glyph. I know that this can be confusing but this is how I managed to get it together.

Just for reference a comparison between LTR and RTL:
```
kerningLTR = {
	m01 = {
		"@MMK_L_A_R" = {
			"@MMK_R_B_L" = -20;
		};
	};
};
kerningRTL = {
	m01 = {
		"@MMK_R_alef_L" = {
			"@MMK_L_beh_R" = -50;
		};
	};
};
```